docs/api/users-delete-user-by-id.md

```diff
@@ -0,0 +1,54 @@
+
+
```

Comment:
```suggestion
```

Reply:
I'm not sure what the minus means in this context; what should I remove?

Reply:
There's nothing you need to do. The "-" is part of the difference display and not your code.
It means that in the suggested change, the line is removed.
The red tint also indicates a removal.